### DISCUSSION TITLE
superset_frontend_packages_superset-ui-chart-controls - Core Vulnerability

### DISCUSSION THREAD

**Discussion by ErrolBlake** [apache/superset#24042] [Q&A / Help]
We performed a dependency track against superset chart 0.9.4 using Cyclondex. The scan found a medium-severity vulnerability related to the core component (relative path to the dependency in the source code: superset-frontend/packages/superset-ui-chart-controls/package.json). The vulnerabilities’ details are referenced in the National Vulnerability Database CVE-2022-43720.

Please let us know if there will be a plan for remediation.

Thank you.

**Comment by dpgaspar** [comment]
Hi https://github.com/advisories/GHSA-fpmr-qmgh-42x2 vulnerability scope is related with a toast message when a user deletes a CSS template with a malicious HTLM tag. Not related with superset-frontend/packages/superset-ui-chart-controls/package.json.
